dotcom-rendering/src/web/components/FixedMediumSlowXIIMPU.tsx

```diff
@@ -129,11 +129,13 @@ const ThreeColumnSliceWithAdSlot = ({
 					))}
 				</UL>
 			</LI>
-			<LI percentage="33.333%" padSides={true} showDivider={true}>
-				<Hide until="tablet">
-					<AdSlot position="inline" index={index} />
-				</Hide>
-			</LI>
+			{trails.length > 0 && (
```

Comment:
Should this check be in place before we [render the container](https://github.com/guardian/dotcom-rendering/pull/7542/files#diff-8191be4c70728a3c48f834c4b6c1865b859f66a1ab2165ba6bfaf88d26504e0aL163-L168) at all?

This would imply that `FixedMediumSlowXIIMPU` can render 3 or 12 cards.

Reply:
Thats a good point. I was hesitant to do that at first because I felt like that was implicitly making a contract that this container can be used to render 3 cards, but I think thats possibly a bit silly considering how little a change this is.